common/hooks/pre-pkg/05-rename-python3-c-bindings.sh

```diff
@@ -0,0 +1,16 @@
+# This hook executes the following tasks:
+#	- renames cpython binding files to not include the arch-specific extension suffix
+
+hook() {
+	if [ ! -d ${PKGDESTDIR}/${py3_sitelib} ]; then
+		return 0
+	fi
+
+	find "${PKGDESTDIR}/${py3_sitelib}" -type f -executable -iname '_[A-Za-z]*.cpython*.so' \
```

Comment:
the regex doesn't work for e.g. borg, should probably be `[A-Za-z_]*.cpython*.so` ?